addons/httprox/common/src/main/java/org/commonjava/indy/httprox/handler/ProxyAcceptHandler.java

```diff
@@ -91,6 +94,9 @@
     @Inject
     private GoldenSignalsMetricSet sliMetricSet;
 
+    @Inject
+    private ProxyTransfersExecutor proxyExecutor;
```

Comment:
Minor point, but I think we can put the following directly in here:

```
   @Inject
    @WeftManaged
    @ExecutorConfig( named = "mitm-transfers", threads = 0, priority = 5, loadSensitive = FALSE )
    private ExecutorService executor;
```

This would reduce the complexity a little bit. It's not critical, but would be good to make it consistent with other executor injection points.

Reply:
That was my initial attempt.
 But I found the object `ProxyAcceptHandler` is missing the `@ApplicationScoped` annotation. The @Inject @WeftManaged and @ExecutorConfig were completely ignored. I can double check if you like ?

Reply:
Ah, ok. I'm not sure whether using @ApplicationScoped would be appropriate here, but I suspect that would be fine. But if this approach works, I'm ok with it too.

Reply:
@ApplicationScoped isn't appropriate on `ProxyAcceptHandler`. It is referenced by the ChannelListeners object. A separate instance for each XNIO worker thread. Those are not application scoped.
 This solution works.